packages/cli/src/config/settingsSchema.ts

```diff
@@ -623,13 +623,13 @@ const SETTINGS_SCHEMA = {
     showInDialog: false,
     properties: {
       fileName: {
-        type: 'object',
+        type: 'string',
```

Comment:
is this accurate? the type is String but seems like it takes a string | string[]?

Reply:
The casting happens in the config, so this doesn't get used, but I refactored this to make it better.